app/templates/_package.json

```diff
@@ -65,6 +65,9 @@
     "generator-jhipster": "<%= packagejs.version %>",
     "lodash": "3.3.1",<% if (buildTool == 'maven') { %>
     "xml2js": "0.4.5",<% } %>
+    "yo": ">=1.3.0",
+    "requirejs": "2.1",
+    "jasmine-core": "~2.1.0",
```

Comment:
We've already had many issues with the "tilde", can you fix the dependency instead?